freqtrade/freqtradebot.py

```diff
@@ -1006,7 +1030,7 @@ def _notify_sell(self, trade: Trade, order_type: str) -> None:
             'exchange': trade.exchange.capitalize(),
             'pair': trade.pair,
             'gain': gain,
-            'limit': trade.close_rate_requested,
+            'limit': profit_rate,
```

Comment:
note to myself: verify this change separately

Reply:
Thanks for feedback.